msteams-platform/webhooks-and-connectors/how-to/connectors-creating.md

```diff
@@ -180,6 +180,25 @@ The following manifest.json file contains the basic elements needed to test and
 }
 ```
 
+## Enable connectors in Teams
```

Comment:
I think "Disable or enable connectors in Teams" is more appropriate as connectors are enabled by default for all tenants except GCC.